libc/shared/str_to_float.h

```diff
@@ -0,0 +1,36 @@
+//===-- String to float conversion utils ------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SHARED_STR_TO_FLOAT_H
+#define LLVM_LIBC_SHARED_STR_TO_FLOAT_H
+
+#include "src/__support/str_to_float.h"
+
+namespace LIBC_NAMESPACE::shared {
+
+using internal::ExpandedFloat;
+using internal::RoundDirection;
```

Comment:
@ldionne @michaelrj-google Since we're using `using declaration` here would it make sense to do that for the function declarations too?

Reply:
that works for me